packages/react-scripts/config/botifyConfig.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const autoprefixer = require('autoprefixer');
+
+const postCssOptions = {
+  // Necessary for external CSS imports to work
+  // https://github.com/facebookincubator/create-react-app/issues/2677
+  ident: 'postcss',
+  plugins: () => [
+    require('postcss-flexbugs-fixes'),
+    autoprefixer({
+      browsers: [
+        '>1%',
+        'last 4 versions',
+        'Firefox ESR',
+        'not ie < 9', // React doesn't support IE8 anyway
+      ],
+      flexbox: 'no-2009',
+    }),
+  ],
+};
+
+module.exports = {
+  babelPresets: [require.resolve('babel-preset-stage-0')],
+  babelPlugins: [require.resolve('babel-plugin-transform-decorators-legacy')],
```

Comment:
can't we use this one https://babeljs.io/docs/plugins/transform-decorators/ ?

Reply:
No we can't, it's not yet supported:
![image](https://user-images.githubusercontent.com/14129459/30959830-9e9f2f64-a441-11e7-9ed4-8c68731da9d4.png)